tests/conftest.py

```diff
@@ -2,12 +2,51 @@
 from requests import HTTPError
 import dash
 from selenium.webdriver.chrome.options import Options
-from selenium.common.exceptions import TimeoutException
-
+from selenium.common.exceptions import TimeoutException, WebDriverException
+from selenium import webdriver
 
 from tests.data.snake_oil_data import ensembles_response
 
 
+def validate_chromedriver():
+    try:
+        driver = webdriver.Chrome()
+        browser_version = driver.capabilities["browserVersion"]
+        chromedriver_version = driver.capabilities["chrome"][
+            "chromedriverVersion"
+        ].split(" ")[0]
+        return browser_version[0:3] == chromedriver_version[0:3]
+    except (WebDriverException, TypeError):
+        return False
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--skip-browser-tests",
+        action="store_true",
+        default=False,
+        help="This option allows skipping tests that depend on chromedriver",
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "browser_test: mark test as chromedriver dependent"
+    )
+
+
+def pytest_collection_modifyitems(config, items):
+    skip_browser_tests = pytest.mark.skip(
+        reason="chromedriver missing in PATH or intentionally skipped"
+    )
+    valid_chromedriver = validate_chromedriver()
+    browser_tests = [item for item in items if "browser_test" in item.keywords]
+    if valid_chromedriver and not config.getoption("--skip-browser-tests"):
```

Comment:
We probably want the case where the chromedriver version is invalid and `--skip-browser-test` is not given to produce an error message. This is so that tests are not silently skipped unintentionally.

Reply:
Hello Eivind, thanks for the comment, I've just pushed the changes. Have a nice day! :)